src/App/components/ClearCompleted.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledClearBtn = styled.button`
+  float: right;
+  position: relative;
+  line-height: 20px;
+  text-decoration: none;
+  cursor: pointer;
+
+  &:active {
+    float: right;
+    position: relative;
+    line-height: 20px;
+    text-decoration: none;
+    cursor: pointer;
+  }
+`
```

Comment:
Considering using flex-box instead of float for all components

Reply:
I fixed this

Reply:
I didn't see the change of code yet.

Reply:
Sorry due to blackout, I can't push the code up. I now can do it